contracts/modules/capital/Pool.sol

```diff
@@ -58,95 +76,107 @@ contract Pool is IPool, MasterAware, ReentrancyGuard {
   }
 
   constructor (
-    address[] memory assetAddresses,
-    uint8[] memory assetDecimals,
-    uint104[] memory _minAmounts,
-    uint104[] memory _maxAmounts,
-    uint16[] memory _maxSlippageRatios,
     address _master,
     address _priceOracle,
-    address _swapOperator
+    address _swapOperator,
+    address DAIAddress,
+    address stETHAddress
   ) {
+    master = INXMMaster(_master);
+    priceFeedOracle = IPriceFeedOracle(_priceOracle);
+    swapOperator = _swapOperator;
 
-    // First asset is ETH
-    assets.push(Asset(ETH, 18, false));
-
-    require(assetAddresses.length == _minAmounts.length, "Pool: Length mismatch");
-    require(assetAddresses.length == _maxAmounts.length, "Pool: Length mismatch");
-    require(assetAddresses.length == _maxSlippageRatios.length, "Pool: Length mismatch");
-    require(assetAddresses.length == assetDecimals.length, "Pool: Length mismatch");
-
-    for (uint i = 0; i < assetAddresses.length; i++) {
+    // The order of payoutAssets should never change between updates. Do not remove the following
+    // lines!
+    payoutAssets.push(Asset(ETH, 18));
+    payoutAssets.push(Asset(DAIAddress, 18));
```

Comment:
This might be error prone as it will be checked manually. Would be better to have a mechanism that will read the values from the previous pool, similar to how we copy over MCR data. Probably doesn't make sense to implement that now because the interface changes, leave a todo note?